packages/store/addon/-private/system/core-store.ts

```diff
@@ -1393,7 +1392,8 @@ abstract class CoreStore extends Service {
       // will once again convert the records to snapshots for adapter.findMany()
       let snapshots = new Array(totalItems);
       for (let i = 0; i < totalItems; i++) {
-        snapshots[i] = internalModels[i].createSnapshot(optionsMap.get(internalModel));
+        let internalModel = internalModels[i];
```

Comment:
Why the multiline?

Reply:
this was a bug that the upgraded lint and typescript tooling caught. internalModel is always undefined currently. Since we need to access it twice I cached it off.